src/python/tensorflow_cloud/core/tests/unit/containerize_test.py

```diff
@@ -31,15 +32,10 @@
 
 class TestContainerize(unittest.TestCase):
     def setup(self):
-        self.test_data_path = os.path.join(
-            os.path.dirname(os.path.abspath(__file__)), "../testdata/"
-        )
-        self.entry_point = os.path.join(
-            self.test_data_path, "mnist_example_using_fit.py"
-        )
+        self.entry_point = "sample.py"
```

Comment:
Would have been better to do /foo/bar/sample.py to test that if it works both absolute path and relative path. In current impl, it isn't entirely clear if this *has* to be absolute path, or can be both absolute or relative.

Reply:
Missed answering this question, sorry about that. This cannot be an absolute path today because of preprocessing and containerizing constraints.

Reply:
It looks like a major contract. Is this relative to where? and where is this specification/constraints enforced? In other words, would unit test fail if this was absolute path, or would it only manifest in integration test where the containarized program is actually invoked?